app/views/external_users/claims/misc_fees/litigators/_misc_fee_fields.html.haml

```diff
@@ -1,42 +1,26 @@
 .form-section.misc-fee-group.nested-fields.js-block.fx-do-init.fx-fee-group.fx-numberedList-item{ data: { 'type': 'miscFees', autovat: @claim.apply_vat? ? 'true' : 'false' } }
 
-  %fieldset
-    %legend
-      %span.govuk-heading-m
-        = t('.misc_fee')
-        %span.fx-numberedList-number
+  = f.govuk_fieldset legend: {} do
```

Comment:
Can we remove empty `legend` option?

Reply:
TODO

Reply:
If you add a meaningful description, this may either create duplicate content or break the layout.

An alternate solution would be to:

- add meaningful text to the `legend` option
- remove your heading span

```
= f.govuk_fieldset legend: { text: t('.misc_fee_html') } do
  = link_to_remove_association
```

Reply:
Done ✅